ragstack-e2e-tests/e2e_tests/langchain/test_compatibility_rag.py

```diff
@@ -137,21 +156,30 @@ def _bedrock_chat(**kwargs) -> BedrockChat:
 
 @pytest.fixture
 def bedrock_anthropic_claudev2_llm():
-    return _bedrock_chat(
-        model_id="anthropic.claude-v2",
-    )
+    return {
+        "llm": _bedrock_chat(
+            model_id="anthropic.claude-v2",
+        ),
+        "nemo_config": None,
```

Comment:
why not?

Reply:
Just untested. Will follow up to test and add these, but wanted to get the OpenAI compatibility in today